src/nvim/tui/tui.c

```diff
@@ -1236,18 +1236,6 @@ static int unibi_find_ext_str(unibi_term *ut, const char *name)
   return -1;
 }
 
-static int unibi_find_ext_bool(unibi_term *ut, const char *name)
```

Comment:
This isn't an exact copy of `unibi_find_ext_str`.  Note how one calls `unibi_count_ext_bool`/`unibi_get_ext_bool_name` while the other calls `unibi_count_ext_str`/`unibi_get_ext_str_name`?